components/calculation/MeasureHighlightingPanel.tsx

```diff
@@ -12,20 +18,146 @@ const useStyles = createStyles({
   panel: {
     maxHeight: '100%',
     overflow: 'scroll'
+  },
+  highlightRed: {
+    backgroundColor: '#edd8d0',
+    color: '#a63b12',
+    borderBottomColor: '#a63b12',
+    borderBottomStyle: 'double'
+  },
+  highlightGreen: {
+    backgroundColor: '#ccebe0',
+    color: '#20744c',
+    borderBottomColor: '#20744c',
+    borderBottomStyle: 'solid'
```

Comment:
For these colors, we probably want to use the pre-build Mantine theme colors to give the app a uniform look. You can pass in `theme` as an argument to the createStyles function, then access `theme.colors.red[x]` or `theme.colors.green[x]`, where x is a number from 1-10 determining the darkness of the color, which will supply an array of colors available in the app.

Reply:
Ahh, I see that these were selected to match the measure highlighting colors, which we don't have access to change in testify, so disregard this comment :)